backend/tournesol/models/entity.py

```diff
@@ -200,10 +200,10 @@ def update_n_ratings(self):
         )
         self.save(update_fields=["rating_n_ratings", "rating_n_contributors"])
 
-    def auto_remove_from_rate_later(self, poll, user) -> None:
+    def auto_remove_from_rate_later(self, poll, user, max_) -> None:
```

Comment:
`max_` is not a very helpful name. Why not `auto_remove_threshold`?

Reply:
> max_ is not a very helpful name. Why not auto_remove_threshold?

`auto_remove_threshold` and `threshold` are indeed better, thanks for the suggestion.

> Second small question: Why reading the settings value outside of the method and passing it as argument rather than reading it from inside the method?

It's an habit, I try to write more or less pure methods to make them easily testable. But in our case, there is no benefit of using any additional parameter indeed.

Reply:
I'm fine with any decision to that regard. Already approved the change 👍  Thanks!

Reply:
Alright, I took the time to move the logic from the view to the model  to follow the Django's "big models small views" mindset. It also simplifies the tests as now they are gatherer in only one file.